src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -162,7 +162,7 @@ export function CurrencySearch({
   const tokenComparator = useTokenComparator(false)
 
   // if they input an address, use it
-  const isAddressSearch = isAddress(debouncedQuery)
+  const isAddressSearch = useMemo(() => isAddress(debouncedQuery), [debouncedQuery])
```

Comment:
`useMemo` is worthless here because `isAddressSearch` is already a primitive type.

Reply:
Agree.

Reply:
Fixed.